utils/database.js

```diff
@@ -126,74 +124,74 @@ class Database {
+                      style += ' 100%);';
                       break;
-                    case "VISHRAAM_COLORED":
+                    case 'VISHRAAM_COLORED':
                     default:
-                      style += " color:";
+                      style += ' color:';
                       style +=
-                        vishraamPositions[index] === "v" ? "#c0392b;" : "#ffc500;";
+                        vishraamPositions[index] === 'v'
+                          ? '#c0392b;'
+                          : '#ffc500;';
                   }
-                  return (
-                    `<span style='
+                  return `<span style='
                     ${style} white-space: nowrap;'>
-                    ${word} </span>`
-                  );
-                } 
-                  if (larivaar && larivaarAssist && index % 2 !== 0) {
-                    style += " opacity: .65;";
-                  }
-                  return `<span style='${  style  } white-space: nowrap;'>${  word  }</span>`;
-                
+                    ${word} </span>`;
+                }
+                if (larivaar && larivaarAssist && index % 2 !== 0) {
+                  style += ' opacity: .65;';
+                  return `<span style='${style} white-space: nowrap;'>${word}</span>`;
+                }
+                return word;
               });
-              let curGurmukhi = larivaar ? arr.join("&#8203;") : arr.join(" ");
+              let curGurmukhi = larivaar ? arr.join('&#8203;') : arr.join(' ');
 
               const translationJson = JSON.parse(row.Translations);
               // Remove nulls
               row.English =
-                translationJson == null || translationJson.en.bdb == null ?
-                " " :
-                translationJson.en.bdb;
+                translationJson == null || translationJson.en.bdb == null
+                  ? ' '
+                  : translationJson.en.bdb;
               row.Punjabi =
-                translationJson == null || translationJson.pu.bdb == null ?
-                " " :
-                translationJson.pu.bdb;
+                translationJson == null || translationJson.pu.bdb == null
+                  ? ' '
+                  : translationJson.pu.bdb;
               row.Spanish =
-                translationJson == null || translationJson.es.sn == null ?
-                " " :
-                translationJson.es.sn;
+                translationJson == null || translationJson.es.sn == null
+                  ? ' '
+                  : translationJson.es.sn;
               let translit = getTranslitText(row.Transliterations, language);
-              translit = translit === "" || translit == null ? " " : translit;
+              translit = translit === '' || translit == null ? ' ' : translit;
 
               if (
                 (baniId === 9 || baniId === 21) &&
-                padcched === "MAST_SABH_MAST"
+                padcched === 'MAST_SABH_MAST'
```

Comment:
Veerji, please refrain from using plain text strings. As mentioned in the above comment you can put this in a constant file and then use it.

Reply:
fixed